### ISSUE TITLE
User catalog

### ISSUE THREAD

**Issue by nullxDEADBEEF** [streamchaser/streamchaser#439]
### Is your feature request related to a problem? Please describe.

It's annoying that I have to leave the page when I find a media that I find interesting and want to save for later.

### Describe the solution you'd like

I want to be able to save lists of media that I have watched, want to watch, and liked. Possibly be able to share them with people around me.

**Comment by Primdahl26** [comment]
You thinking a store or actually implementing support for users?

**Comment by nullxDEADBEEF** [comment]
Well initially it could be a store, though I'm not sure how well its gonna function in a long run?

**Comment by Primdahl26** [comment]
Yeah i agree a login system would be dope, think we should have a meeting about how we could implement it

**Comment by nullxDEADBEEF** [comment]
Yeah for sure!

**Comment by AndreasPB** [comment]
The idea (so far) is using [EdgeDB](https://www.edgedb.com/) and OAuth 2.0 through Google (maybe more)

**Comment by Primdahl26** [comment]
Were getting closer and closer... We now have sign in with google, and a lot of the infrastructure to making watch lists. We need to finish the frontend for the profile page, and a few things with the API.

1. HTTP Status koder fra POST/ DELETE endpoints (Hvis de faar daarlig streamchaser_id, eller ikke findes i watch listen)
2. HTTP status koder i vores jwt middleware
3. Refreshing af jwt

**Comment by AndreasPB** [comment]
We have been doing a lot of work on this and it is currently in a working state behind a feature flag etc. #449 and a lot of other PRs
